video_xblock/static/css/brightcove.css

```diff
@@ -326,4 +326,11 @@ body {
     color: inherit;
     background: rgba(115,133,159,.5);
     border-radius: 0;
-}
\ No newline at end of file
+}
+/* styles for theme */
+.bc-player-default_default .vjs-big-play-button:active,
```

Comment:
Please add a comment about `default_default` and Brightcove's themes.
So we can find this in the future.